include/roboteam_ai/stp/PlayDecider.hpp

```diff
@@ -25,6 +25,7 @@ class PlayDecider {
      * @param play Play to lock to
      */
     static void lockPlay(Play *play);
+    static bool change;
```

Comment:
What I would suggest is having a Play* here, which if it's nullptr is not selected from the interface. That way in decidePlay you can just

```cpp
if (interfacePlay) {
    currentPlay = interfacePlay;
   // update n stuff...
}
```

Not exactly sure what the `change` stands for either. Does it mean the internal AI can change the play? does it mean the interface has selected a play that should be changed to?... if so, where is this play stored? 

Where does the playdecider check whether a play is available from the interface?